monai/transforms/spatial/array.py

```diff
@@ -553,7 +553,7 @@ def __call__(
             input_affine = np.eye(sr + 1, dtype=np.float64)
         affine_ = to_affine_nd(sr, convert_data_type(input_affine, np.ndarray)[0])
 
-        out_d = self.pixdim[:sr]
+        out_d = self.pixdim[:sr].copy()
```

Comment:
Please confirm whether we need `deepcopy` here.

Thanks.

Reply:
yes, that should be fine given self.pixdim a numpy array.